scripts/batch-score.coffee

```diff
@@ -0,0 +1,50 @@
+# Description:
```

Comment:
A note on commit message: It provides no information at all. @aseem09 could you change it to something which tells us `why` we are doing `what` we are doing? Something like:
```
scripts: add batch-score.coffee

Our bot already has a command `bot score` which tells us the `score`
of the user, i.e., the amount of appreciation received. But let's say
we want to know the score of a whole batch, we'll need to manually
enter the name of each member, which is quite a tedious task. Add
`batch-score` script to make this task easier.
```

Tweak this message if you like :wink:.
NB: the message is wrapped at around 72 chars, which makes `git log` to indent it beautifully.